src/modules/savemanager/savemanager-service.js

```diff
@@ -403,6 +409,37 @@ angular.module('anol.savemanager')
                 );
             };
 
+            /**
+             * Check if feature was updated on remote.
+             * @param {number} featureId The id of the feature.
+             * @param {anol.layer.Feature} layer The layer the feature belongs to.
+             * @returns True, if the feature was updated on remote. False otherwise.
+             */
+            SaveManager.prototype.hasFeaturePollingChanges = function (featureId, layer) {
+                if (!layer) {
+                    return false;
+                }
+                const pollingResult = this.lastPollingResults[layer.name];
+                if (!pollingResult || !pollingResult.length) {
+                    return false;
+                }
+
+                const feature = layer.getFeatures().find(f => f.getId() === featureId);
+                if (!feature) {
+                    return false;
+                }
+
+                return pollingResult.some(pollingItem => {
+                    if (pollingItem.id !== featureId) {
+                        return false;
+                    }
+
+                    var pollingModified = new Date(pollingItem.modified);
+                    var featureModified = new Date(feature.get('modified'));
+                    return pollingModified > featureModified;
+                });
```

Comment:
just an alternative:

```suggestion
                const feature = layer.getFeatures().find(f => f.getId() === featureId);
                const pollingItem = pollingResult.find(pollingItem => pollingItem.id === featureId);
                
                if (!feature || !pollingItem) {
                    return false;
                }

                var pollingModified = new Date(pollingItem.modified);
                var featureModified = new Date(feature.get('modified'));
                return pollingModified > featureModified;
```